agent.md

```
@@ -0,0 +1,1268 @@
+
+**Infrastructure:**
+- Docker (PostgreSQL, Redis, Jaeger)
+- Git (version control)
+
+---
+
+## Architecture
+
+### System Architecture
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)